.travis.yml

```diff
@@ -1,10 +1,17 @@
 language: generic
 
+env:
+  - MYSQL_IMAGE=mysql:5.6
+  - MYSQL_IMAGE=mysql:5.7
+  - POSTGRES_IMAGE=postgres:9.5
+  - POSTGRES_IMAGE=postgres:9.6
+  - POSTGRES_IMAGE=postgres:10.4
```

Comment:
... won't these overwrite each other?

Reply:
I thought that in the first case as well but travis sees that there are multiple of the same variable so it will run a container for every variable. See attached image. <a href="https://docs.travis-ci.com/user/speeding-up-the-build/">Speeding up the build</a>

Reply:
Wow, what a strange syntax to produce that behavior. 

![image](https://user-images.githubusercontent.com/208647/42049966-e1ea5028-7ac4-11e8-93ec-47416e30a019.png)